lib/common-types.ts

```diff
@@ -343,6 +353,12 @@ type MRInternal<UA extends boolean> = {
   (opts: MROpts<'Release', 'create', UA>): MRReturn<'Release', 'create'>
   (opts: MROpts<'Release', 'update', UA>): MRReturn<'Release', 'update'>
   (opts: MROpts<'Release', 'delete', UA>): MRReturn<'Release', 'delete'>
+  (opts: MROpts<'Release', 'publish', UA>): MRReturn<'Release', 'publish'>
+  (opts: MROpts<'Release', 'unpublish', UA>): MRReturn<'Release', 'unpublish'>
+  (opts: MROpts<'Release', 'validate', UA>): MRReturn<'Release', 'validate'>
+
+  (opts: MROpts<'ReleaseAction', 'get', UA>): MRReturn<'ReleaseAction', 'get'>
+  (opts: MROpts<'ReleaseAction', 'query', UA>): MRReturn<'ReleaseAction', 'query'>
```

Comment:
I'm wondering if we should call this something that indicates it's for a particular release (and not all release actions in general). Perhaps `queryForRelelease`?

Reply:
I'm in favour of `release.getAction`, and `release.queryAction`

Reply:
I'm ok with `queryForRelease` 👍🏽 

@mshaaban0 they are normally exposed as such `client.releaseAction.get()`, so maybe `releaseAction.getAction` might be a bit verbose? 🤔